modules/contentbox/modules/contentbox-admin/views/widgets/editorSelectorHelper.cfm

```diff
@@ -76,6 +76,10 @@ function switchWidgetFormMode( mode ) {
                 iconName = detail.find( "##widgetIcon" ).val();
                 titleBar.html( '<i class="fa fa-' + iconName + '"></i> Insert \'' +widgetDisplayName+ '\' Widget' );
                 //updatePreview();
+
+                if(widgetName == 'Menu' && !$('##widget-preview-content li').length){
```

Comment:
Could we make this generic, where it tests ANY widget?

Reply:
I would say only check for the widget preview content, so change it to only this for future proofing it:

```
if( !$('##widget-preview-content li').length ){
```